src/containers/profile/coursesTab/courseCard.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import { Link } from 'react-router-dom';
```

Comment:
Check the order of importing the components and libs. Ask Slava about the right order of import